gd32e10x/standard_peripheral/include/gd32e10x_i2c.h

```diff
@@ -315,6 +315,12 @@ typedef enum
 #define I2C_ADDFORMAT_7BITS           ((uint32_t)0x00000000U)                  /*!< address:7 bits */
 #define I2C_ADDFORMAT_10BITS          I2C_SADDR0_ADDFORMAT                     /*!< address:10 bits */
 
+/* I2C clock frequency, MHz */
+#define I2CCLK_MAX                    ((uint32_t)0x0000003FU)                  /*!< i2cclk maximum value */
+#define I2CCLK_SM_MIN                 ((uint32_t)0x00000002U)                  /*!< i2cclk minimum value for standard mode */
+#define I2CCLK_FM_MIN                 ((uint32_t)0x00000008U)                  /*!< i2cclk minimum value for fast mode */
+#define I2CCLK_FM_PLUS_MIN            ((uint32_t)0x00000018U)                  /*!< i2cclk minimum value for fast mode plus */
```

Comment:
note: this information could likely be encoded directly in the Zephyr driver, some drivers (e.g. GPIO/PWM already add extra definitions not provided by HAL).

Reply:
The problem is I2CCLK_MAX have different value for GD32 SoCs. If we put these to i2c-gd32 driver, It wil looks like below:
```
+#define I2CCLK_MAX                    ((uint32_t)0x0000003FU)

+#define I2CCLK_MAX                    ((uint32_t)0x0000003FU)
```